src/server-main.js

```diff
@@ -188,12 +188,11 @@ function sanitizeStringArg(val) {
  * @throws
  */
 async function parsePort(host, strPort) {
-	let specificPort;
 	if (strPort) {
 		let range;
 		if (strPort.match(/^\d+$/)) {
-			specificPort = parseInt(strPort, 10);
-			if (specificPort === 0) {
+			const specificPort = parseInt(strPort, 10);
+			if (specificPort) {
```

Comment:
I [see](https://github.com/microsoft/vscode/issues/161112) people use `--port 0` , is value zero allowed?

Reply:
yes `0` is for random port